application/server/server_smoke_test.go

```diff
@@ -73,16 +73,17 @@ func Test_SmokeWorkspaceScan(t *testing.T) {
 			file2:                codeFile,
 			useConsistentIgnores: false,
 			hasVulns:             true,
+			endpoint:             endpoint,
 		},
-		{
-			name:                 "OSS and Code with V1 endpoint",
-			repo:                 "https://github.com/snyk-labs/nodejs-goof",
-			commit:               "0336589",
-			file1:                ossFile,
-			file2:                codeFile,
-			useConsistentIgnores: false,
-			endpoint:             path.Join(endpoint, "/v1"),
-		},
+		//{
```

Comment:
I don't think these tests ever did what we thought they were meant to, since the endpoint wasn't getting passed down to `runSmokeTest`. The result is that we're not computing the `SNYK_API` well when the endpoint is `https://app.snyk.io/api/v1`I don't think these tests ever did what we thought they were meant to, since the endpoint wasn't getting passed down to `runSmokeTest`. The result is that we're not computing the `SNYK_API` well when the endpoint is `https://app.snyk.io/api/v1`